src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseTransport.php

```diff
@@ -24,16 +24,20 @@
 
 /**
  * @author Jeroen Spee <https://github.com/Jeroeny>
+ * @author Cesur APAYDIN <https://github.com/cesurapp>
  */
 final class FirebaseTransport extends AbstractTransport
 {
-    protected const HOST = 'fcm.googleapis.com/fcm/send';
+    protected const HOST = 'fcm.googleapis.com/v1/projects/project_id/messages:send';
```

Comment:
Looking at all the other Notifier transports, there seems to be almost none that use a full URL in the `HOST` constant – they mostly only have the host in there and build the necessary project URL in the `doSend` method. Not sure if that's really relevant, but it would prefent `$host` from always being set and the `HOST` constant from being an acutally invalid URL.